src/components/workspace-utils.js

```diff
@@ -184,6 +185,11 @@ export const SnapshotInfo = ({
         h(SnapshotLabeledInfo, { title: 'Creation Date:', text: Utils.makeCompleteDate(createdDate) }),
         div({ style: { ...Style.elements.sectionHeader, marginBottom: '0.2rem' } }, ['Description:']),
         div([sourceDescription]),
+        h(SnapshotLabeledInfo, {
+          title: 'Data Repo Snapshot Id:', text: [h(Link, {
```

Comment:
I recommend you try out navigating to this link with a screen reader ([VoiceOver instructions](https://webaim.org/articles/voiceover/)). Right now all you hear is the snapshotID, which is unlikely to meaningful. Here are suggestions on how to improve this-- https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA8.

Also, in the future please add steps on how to get to the modified UI elements (a screenshot is also helpful).

Reply:
Thanks!  I've added just a description of where the link goes - is that enough or should it also read out the id?

Reply:
I don't think it's necessary to include the ID-- it seems unlikely that it will be meaningful, and if the user really wants to know what it is, they can use VO keys (VO-Shift-Down) to navigate into the link and get the details.